keras/src/layers/layer.py

```diff
@@ -289,6 +291,9 @@ def __init__(
         self._convert_input_args = True
         # Whether to allow non-tensors as positional arguments in `call()`.
         self._allow_non_tensor_positional_args = False
+        if backend.backend() == "openvino":
```

Comment:
Hi @fchollet, please advice how to avoid this code. Keras common requires it to be of `KerasTensor` type during calling op-by-op. How to relax this check and allow to pass ov type object into ops wrappers?

Reply:
All you need is for `backend.is_tensor(x)` to return True for the types you consider to be tensors.

Reply:
@fchollet, I redefined it for my backend but `openvino.runtime.Output` does not have `dtype` attribute and exception is raised here: https://github.com/keras-team/keras/blob/master/keras/src/dtype_policies/dtype_policy.py#L200
We retrieve dtype using method `get_element_type`. I think `backend.standardize_dtype` should accept `x` instead of `x.dtype`.

Reply:
Removed